openapi-cli/src/test/resources/multipart_formdata.yaml

```diff
@@ -0,0 +1,36 @@
+openapi: "3.0.0"
```

Comment:
resources/utils/swagger contains the same file. Is it intentional?

Reply:
No. Removed in https://github.com/ballerina-platform/openapi-tools/commit/b073cda18ba34b0a9e050f7cc820da3107ffa33b